platforms/hyperledger-indy/charts/indy-ledger-txn/templates/job.yaml

```diff
@@ -0,0 +1,62 @@
+apiVersion: batch/v1
```

Comment:
The pool genesis config map has to be present inside the job pod, otherwise it won't work. Have a look at this

https://github.com/hyperledger-labs/blockchain-automation-framework/blob/9cad77b8bad2a6ee461b3bde125d6dc4d0459e1b/platforms/hyperledger-indy/charts/indy-node/templates/statefulset.yaml#L167

https://github.com/hyperledger-labs/blockchain-automation-framework/blob/9cad77b8bad2a6ee461b3bde125d6dc4d0459e1b/platforms/hyperledger-indy/charts/indy-node/templates/statefulset.yaml#L162